Resources/Prototypes/Procedural/salvage_mods.yml

```diff
@@ -175,10 +195,15 @@
   cost: 4
   desc: Toxic atmosphere
   gases:
-  - 0
-  - 0
-  - 0
-  - 103.927899 # plasma
+  - 0 # oxygen
+  - 21.824779 # nitrogen
+  - 21.824779 # carbon dioxide
+  - 38.453562 # plasma
+  - 0 # tritium
+  - 0 # water vapor
+  - 0 # miasma
```

Comment:
with nerfed miasma, do miasma instead of nitrous. we don't have a nitrous source yet